posts/interaction-media-features.md

```diff
@@ -0,0 +1,11 @@
+feature: Interaction media features (hover, pointer, any-hover, any-pointer)
+status: caution
+tags: polyfill
+kind: css
+polyfillurls: [mq4-hover-hover-shim](https://github.com/cvrebert/mq4-hover-hover-shim)
```

Comment:
Maybe http://www.jordanm.co.uk/post/66660448615/potential-use-cases-for-script-hover-and-pointer ?